src/structurizr/model/element.py

```diff
@@ -100,7 +100,11 @@ def get_afferent_relationships(self) -> Iterator[Relationship]:
         )
 
     def add_relationship(
-        self, relationship: Optional[Relationship] = None, **kwargs
+        self,
+        relationship: Optional[Relationship] = None,
+        *,
+        create_implied_relationships: bool = True,
```

Comment:
I'm wondering if it is advisable to have the default be true. It might be rather surprising behaviour. The Java `Element.addRelationship` doesn't have this argument but I agree that it makes sense to have here. Except maybe the default should be false?

Reply:
I see now that the default strategy is not set (or does nothing). So true is fine, I guess. I blame the GitHub file order :stuck_out_tongue_closed_eyes: